Classes/Hooks/Interfaces/TimeSpan.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OliverKlee\Seminars\Hooks\Interfaces;
+
+/**
+ * Use this interface for hooks concerning the date span.
```

Comment:
This should be consistent with the class name (or the comment should explain the difference).